Backend.Core/ItemsManagers/LayersManager.cs

```diff
@@ -2,33 +2,65 @@
 using Backend.Models.RoomItems;
 using System.Collections.Generic;
 using System.Linq;
+using Serilog;
 namespace Backend.Core
 {
     internal class LayersManager : IRoomItemsManager
     {
-        private readonly Dictionary<string, Layer> _layers = new Dictionary<string, Layer>();
+        //private readonly Dictionary<string, Layer> _layers = new Dictionary<string, Layer>();
+        private List<Layer> _layers = new List<Layer>();
```

Comment:
Not sure, if it is okay. If you want to sort them maybe just add index field to Layer class?  But I can accept List also..

Reply:
I don't think it is worth to stick with Dictionary in this case. Layers' order is part of their logic, so List is more appropriate, there are also small amount of them, so Dict. indexing doesn't save significant amount of time